shell/client/setup-wizard/wizard.html

```diff
@@ -262,7 +262,7 @@ <h1 class="setup-page-title">Sandstorm for Work feature key</h1>
 
 <template name="setupWizardIdentity">
 <div class="setup-page-content">
-  <h1 class="setup-page-title">Configure Identity Provider(s)</h1>
+  <h1 class="setup-page-title">Identity providers</h1>
```

Comment:
I see this says "Identity providers" so, well, OK. See above. Consistency is priority no. 1. In my little world, we'd call these Login Providers. I can live with either.